src/requirements.txt

```diff
@@ -43,6 +47,7 @@ psycopg2==2.7.3.1
 ptyprocess==0.5.2
 py==1.4.34
 pycodestyle==2.3.1
+pycrypto==2.6.1
```

Comment:
This and pycrptodome not used anywhere last we checked.

Reply:
I'll remove it then